lib/pdf/core/object_store.rb

```diff
@@ -19,6 +19,18 @@ def initialize(opts = {})
 
         @info  ||= ref(opts[:info] || {}).identifier
         @root  ||= ref(Type: :Catalog).identifier
+
+        # PDF/A-1b requirement: XMP metadata
+        @xmp_metadata ||= ref(Type: :Metadata, Subtype: :XML).identifier
```

Comment:
I would like this to be optional. The documents we generate now should not get bigger for no reason with new version of Prawn. Let's include this only in documents that actually have some metadata.